openapi/nexus.json

```diff
@@ -6085,5 +6085,126 @@
         ]
       }
     }
-  }
+  },
+  "tags": [
+    {
+      "name": "disks",
+      "description": "Virtual disks are used to store instance-local data which includes the operating system.",
+      "externalDocs": {
+        "url": "http://oxide.computer/docs/#xxx"
+      }
+    },
+    {
+      "name": "firewall",
+      "description": "Firewall operation controls the flow of network data into a VPC",
+      "externalDocs": {
+        "url": "http://oxide.computer/docs/#xxx"
+      }
+    },
+    {
+      "name": "hidden",
+      "description": "TODO operations that will not ship to customers",
+      "externalDocs": {
+        "url": "http://oxide.computer/docs/#xxx"
```

Comment:
These are "https://docs.oxide.computer/api/{lowercase tag name}" if you want to fill them in with the real url

Reply:
But I don't technnically need that right now anyways but just so you know

Reply:
These aren't links to the API documentation; they're links to the product documentation.

Reply:
Ah ha, I see, like a tutorial? `https://docs.oxide.computer/tutorials/{thing}` ?

Reply:
or the thing in the rack? regarless I gues it doesn't really matter

Reply:
Yeah, like that. e.g. https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instances-and-amis.html

Reply:
ah gotcha!